draft-ietf-taps-interface.md

```diff
@@ -1340,7 +1407,7 @@ In addition, incoming entangled Connections can be received by creating a
 Listener on an existing connection:
 
 ~~~
-Listener := Connection.Listen()
+Listener := Connection.ListenClone()
```

Comment:
I'm wondering if we should a bit more detail about how ListenClone() works.
Is this listening on anything that the original Preconnection has specified (e.g., multiple local interfaces), or is this just listening on the LocalEndpoint where the cloned Connection came in (e.g., just one local interface)?
To make this clear, we could add text like: "ListenClone() creates a Listener that listens on the same LocalEndpoint as the cloned Connection is using. Any new Connection received by this Listener will be entangled with the cloned Connection."

If this is non-trivial, we could also make this another issue or continue discussing in #441.

Reply:
I think your proposed sentence fixes this nicely.